### DISCUSSION TITLE
[Feature] Transcript of the video

### DISCUSSION THREAD

**Discussion by Shinbly** [immich-app/immich#13503] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I was thinking about a feature that might help for the searching with video
The goal is to add the transcript of the video in the metadata of the video. and be able to search for a specific world said in the video. 
for that I was thinking of a new Task that would be **Transcript Video** that would extract the audio from the video and send it to a speesh to text service. It could be Wisper added in the immich machine learning or maybe add a field to provide a url to a wisper instance for exemple.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Related: #3168

**Comment by JD1337** [comment]
Hopefully this will be something possible in workflows, but, would still need to be integrated into search, right now the OCR and Clip search are seperate. Its not scalable to keep adding new search types, there'd need to be some way to dynamically include search from task outputs.
That would open floodgates for things like this.

**Comment by Tschipcraft** [comment]
Duplicate with more info: https://github.com/immich-app/immich/discussions/14658
